lib/game.rb

```diff
@@ -5,7 +5,7 @@ def initialize
     self.grids = [Grid.new]
     loop do
       step!
-      sleep 1
+      sleep 0.1
```

Comment:
Do you think this is the change that causes your computer to hang? Or can you isolate it elsewhere?